clang-tools-extra/clang-tidy/bugprone/StringviewNullptrCheck.cpp

```diff
@@ -74,8 +74,9 @@ RewriteRuleWith<std::string> StringviewNullptrCheckImpl() {
   auto BasicStringViewConstructingFromNullExpr =
       cxxConstructExpr(
           HasBasicStringViewType, argumentCountIs(1),
-          hasAnyArgument(/* `hasArgument` would skip over parens */ anyOf(
-              NullLiteral, NullInitList, EmptyInitList)),
+          hasAnyArgument(
+              /* `hasArgument` would skip over parens */ ignoringParenImpCasts(
+                  anyOf(NullLiteral, NullInitList, EmptyInitList))),
```

Comment:
Checkout this file later for this and the other section I highlighted, here it looks like the only reason that  `hasAnyArgument` was chosen is because of this differing behavior, instead, this can be replaced with `hasArgument(0, ...)` after the `IgnoreParenImpCasts` is removed from the `hasArgument` matcher.